git_pull/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM $BUILD_FROM
 ENV LANG C.UTF-8
 
 # Setup base
-RUN apk add --no-cache jq curl git
+RUN apk add --no-cache jq curl git openssh
```

Comment:
`openssh-client`

Reply:
Sorry for the dumb mistakes and thank you for your patience!